rfcs/rdm-0068-data-migration.md

```diff
@@ -211,8 +215,10 @@ access to the database and application context.
 #### Load
 
 Loading the data into the new InvenioRDM instance is the last step of the
-stream. Data needs to be validated, prepared and inserted into the corresponding
-tables. The preparation step, enables optimizations such as bulk inserts or 
+stream. The input of this step is aligned with the output of the transform step.
```

Comment:
aligned is a bit vague in my opinion since we want to strictly follow the dict API keys.
```diff
- aligned
+ is bounded to the programmatic API 
```